website/app/blog/page.tsx

```diff
@@ -63,7 +63,7 @@ const Home = () => {
   return (
     <Box as="main" p="xl">
       <Box margin="0 auto" maxWidth={600}>
-        <Text color="secondary-red" textAlign="center" variant="subtitle-md">
```

Comment:
cc @theo-mesnil what is this color? 🤔

Reply:
An invention (it's the next colors but it's not ok here :D)

Reply:
@mleralec it's fixed here https://github.com/WTTJ/welcome-ui/pull/2539